.eslintrc.json

```diff
@@ -28,6 +28,7 @@
         "process": true
     },
     "parserOptions": {
+        "ecmaVersion": 2017,
```

Comment:
Probably should be ES6 (2015+) since that's where the codebase is currently at and probably should remain until we make a conscious decision to start utilising >ES6 syntax.